src/main/java/org/apache/commons/lang3/CharEncoding.java

```diff
@@ -23,14 +23,14 @@
 /**
  * <p>Character encoding names required of every implementation of the Java platform.</p>
  *
- * <p>According to <a href="http://docs.oracle.com/javase/1.3/docs/api/java/lang/package-summary.html#charenc">JRE character
+ * <p>According to <a href="https://docs.oracle.com/javase/6/docs/api/java/nio/charset/Charset.html">JRE character
```

Comment:
Javadoc references should be to Java **8** since this is the Java level this component requires.

Reply:
Fixes